### PR TITLE
feat: show cluster versions for database objects

### LINKED ISSUE #13614: Record the original system version + commit ID when users created database objects
### Is your feature request related to a problem? Please describe.

- materialized views
- indexes
- views
- databases
- schemas
- sinks
- tables
- sources

Theoritically, all these objects require recording the versions.

This will allow us to quickly diagnose in which version upgrade the breaking change happened.
cc @stdrc 

### Describe the solution you'd like

_No response_

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by hzxa21**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Fix #13614.

Store `initialized_at_cluster_version` and `created_at_cluster_version` in meta store and display them in rw system tables.

Limitation: the version is based on the meta node version. It may not accurately reflect the compute node version if we are using different codes for different roles, which is unlikely in most cases.

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Comment by hzxa21** [comment]
Example output (risedev d):
```
dev=> select * from rw_tables;
  id  | name | schema_id | owner |       definition       |                 acl                 |        initialized_at         |          created_at           |         initialized_at_cluster_version          |           created_at_cluster_version            
------+------+-----------+-------+------------------------+-------------------------------------+-------------------------------+-------------------------------+-------------------------------------------------+-------------------------------------------------
 1001 | a    |         0 |     1 | CREATE TABLE a (a INT) | {postgres=arwd/root,root=arwd/root} | 2023-12-23 05:16:34.069+00:00 | 2023-12-23 05:16:34.094+00:00 | PostgreSQL 9.5-RisingWave-1.5.0-alpha (unknown) | PostgreSQL 9.5-RisingWave-1.5.0-alpha (unknown)
(1 row)
```

`unkown` should be replaced by the git commit hash in release build.

File: src/meta/model_v2/src/object.rs [hunk 1]
```diff
@@ -66,6 +66,8 @@ pub struct Model {
     pub database_id: Option<DatabaseId>,
     pub initialized_at: DateTime,
     pub created_at: DateTime,
+    pub initialized_at_cluster_version: Option<String>,
```
Comment (yezizp2012) [inline src/meta/model_v2/src/object.rs:69]: We have to add these columns for object table as well in src/meta/model_v2/migration/src/m20230908_072257_init.rs.

**Review by yezizp2012 — COMMENTED** [review]
Rest LGTM!

File: src/meta/model_v2/src/object.rs (hunk 1 above)
Comment (hzxa21) [reply src/meta/model_v2/src/object.rs:69]: Fixed.

**Review by yezizp2012 — APPROVED** [review]

**Pull request merged**